libraries/SITL/SIM_AirSim.cpp

```diff
@@ -148,15 +196,11 @@ void AirSim::recv_fdm()
         return;
     }
 
-    bool parse_ok = parse_sensors((const char *)(p1+1));
+    parse_sensors((const char *)(p1+1));
 
     memmove(sensor_buffer, p2, sensor_buffer_len - (p2 - sensor_buffer));
     sensor_buffer_len = sensor_buffer_len - (p2 - sensor_buffer);
 
-    if (!parse_ok) {
-        return;
-    }
```

Comment:
This was removed since with Lidar, sometimes no data is sent, or incomplete is received, therefore removed this check

Reply:
ok